backend/game.py

```diff
@@ -20,62 +20,24 @@ def __str__(self):
         return str(self.current_board)
 
     def is_game_over(self):
-        pass
+        #for each aura'd hex in a room, check if the linked region has all seven rooms.
+        winners = []
+        for hex in self.current_board.rooms[0].hexes:
+            if hex.aura:
+                linked_names = [x.name for x in linked_rooms(self.current_board, hex)]
+                if all([(x in linked_names) for x in ['P','I','O','U','S','L','Y']]):
```

Comment:
a more normal way to do this would be to compare sets or do:
```suggestion
                if sorted(linked_names) == sorted(['P','I','O','U','S','L','Y']):
```